### PR TITLE
Added banner and moved cookie down

### PR DESCRIPTION
**Pull request by tymeklew**
Added banner for 2024 is live and moved cookie selector down slightly

### CODE REVIEW

**Comment by Pandaroses** [comment]
lgtm

**Comment by shardros** [comment]
Authorised the deployment on Netlify. I think if you take a branch within the repo rather than forking the repo then the build should happen automagically. Which I think you should have perms to do. Will review when build is complete.

File: .vuepress/theme/styles/theme.styl [hunk 1]
```diff
@@ -54,6 +54,26 @@ body
   border-right 1px solid $borderColor
   overflow-y auto
 
+.banner
```
Comment (shardros) [inline .vuepress/theme/styles/theme.styl:57]: Not quite sure why but you do have a small bug that once the cookie notice is accepted there is white space between the new banner and the hero image which looks weird: 
image

File: .vuepress/theme/Banner.vue [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<template>
```
Comment (shardros) [inline .vuepress/theme/Banner.vue:1]: I like this component making.

**Review by shardros — APPROVED** [review]
Well done. This all looks very reasonable to me. You do have a small bug when the cookie notice is accepted then there is a section of whitespace. Something with your CSS I suspect.

File: .vuepress/theme/styles/theme.styl (hunk 1 above)
Comment (tymeklew) [reply .vuepress/theme/styles/theme.styl:57]: ill try to fix that now thx

Comment (tymeklew) [reply .vuepress/theme/styles/theme.styl:57]: should be fixed now just moved the img up a bit

**Comment by tymeklew** [comment]
Missclick didnt mean to dissmiss comment but yeah everythimg should be working now

**Pull request merged**